src/charm.py

```diff
@@ -42,16 +37,26 @@
 )
 from ops.main import main
 from ops.model import ActiveStatus, BlockedStatus, MaintenanceStatus, Relation, WaitingStatus
-from ops.pebble import ChangeError, Error, Layer
+from ops.pebble import Layer
```

Comment:
Unrelated to this PR ID037, but we should import everything from `ops`. e.g. instead of `from ops.model import ActiveStatus` we should have `from ops import ActiveStatus`.

Reply:
I was able to change this for everything apart from the Error, Layer and LayerDict imports